webpack.config.js

```diff
@@ -236,7 +236,7 @@ function getWebpackConfig( { cssFilename, externalizeWordPressPackages = false }
 					loader: 'html-loader',
 				},
 				{
-					test: /\.(svg)$/,
+					test: /\.(?:svg|png)$/i,
```

Comment:
I think we might as well add `jpg` here already now so that the next one who needs it wouldn't need to wonder why it isn't working for them. E.g. Map and latest posts blocks both use jpegs.

Reply:
I was going to start adding a lot but wasn't sure where to draw the line…

`svg|png|gif|jpg|jpeg|ico|png|tiff|webp`

Add em as you need em?

Reply:
> Map and latest posts blocks both use jpegs.

I'll role those changes in here.

Reply:
Just adding `jpg` seems like a pragmatic and useful choice right now.

Reply:
Done, related posts updated 🎉